msg/translation_node/README.md

```diff
@@ -0,0 +1,66 @@
+# Message Translations
+
+This package contains a message translation node and a set of old message conversion methods.
+It allows to run applications that are compiled with one set of message versions against a PX4 with another set of message versions, without having to change either the application or the PX4 side.
+
+Specifically for this to work, topic publication/subscription/service names contain a message version in the form of `<topic_name>_v<version>`.
+
+The translation node knows about all existing message versions, and dynamically monitors the publications, subscriptions and services, and then creates translations as needed.
+
+## Updating a Message
+When changing a message, a new version needs to be added.
+The steps include:
+- Copy the message file (`.msg`/`.srv`) to `px4_msgs_old/msg` (or `px4_msgs_old/srv`) and add the current version to the file name.
+  For example `msg/versioned/VehicleAttitude.msg` becomes `px4_msgs_old/msg/VehicleAttitudeV3.msg`.
+  Update the existing translations that use the current topic version to the now old version.
```

Comment:
```
Update the existing translations that use the current topic version to the now old version.
```
How/where shall this be done?

Reply:
This is now described in more depth in https://github.com/PX4/PX4-user_guide/pull/3465. I'll remove it from here and link to that instead.